### DISCUSSION TITLE
Example of configurations

### DISCUSSION THREAD

**Discussion by phlnil** [blakeblackshear/frigate#11504] [Config Support]
### Describe the problem you are having

Installing Frigate 
to avoid any issue with camera or decode/encoding 
what is the smallest configuration required in 
<CT_ID>.conf
docker-compose.yaml
config.yml 
(so no coral, no detect, passtrhough  no record... barely see the server,
<iP>:5000 working again.

(assuming on last debian/portainer/compose)

thank you!

### Version

no

### Frigate config file

```yaml
no
```

### Relevant log output

```shell
no
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

CPU (no coral)

**Comment by blakeblackshear** [comment]
It's all right in the getting started guide: https://docs.frigate.video/guides/getting_started

**Reply by phlnil** [reply]
Yes, and thank you for all that wonderful job!
I was thinking to  a really simple or even the simplest configuration approach (at the beginning I didn't even see Frigate logo on :5000)
i.e. a config iGPU(no coral), No detectors, No Records, No Events... barely to see it starting up could be useful.
To avoid at least at the beginning to submit configurations with a lot of silly errors for anyone that already is deep in the matter.
Thx.
